packages/fxa-auth-server/test/local/routes/subscriptions/stripe-webhooks.js

```diff
@@ -486,6 +512,101 @@ describe('StripeWebhookHandler', () => {
       });
     });
 
+    describe('handleProductUpdatedEvent', () => {
+      let scopeContextSpy, scopeSpy, captureMessageSpy;
+      beforeEach(() => {
+        captureMessageSpy = sinon.fake();
+        scopeContextSpy = sinon.fake();
+        scopeSpy = {
+          setContext: scopeContextSpy,
+        };
+        sandbox.replace(Sentry, 'withScope', (fn) => fn(scopeSpy));
+        sandbox.replace(Sentry, 'captureMessage', captureMessageSpy);
+      });
+
+      it('throws a sentry error if the update event data is invalid', async () => {
```

Comment:
I think this test would still pass if `handleProductUpdateEvent` returns after validating the first plan.  Not a big deal I suppose, since an implementation like that is unlikely to get past review.  But this test doesn't verify if all the plans were validated.